test/e2e_flags.go

```diff
@@ -73,9 +82,47 @@ func initializeFlags() *EnvironmentFlags {
 
 	flag.StringVar(&f.Tag, "tag", "latest", "Provide the version tag for the test images.")
 
+	klog.InitFlags(nil)
+	flag.Set("v", "2") // A normal level of logging for testing
+	flag.Set("alsologtostderr", "true")
+
 	return &f
 }
 
+func printFlags() {
+	fmt.Print("Test Flags: {")
+	flag.CommandLine.VisitAll(func(f *flag.Flag) {
+		fmt.Printf("'%s': '%s', ", f.Name, f.Value.String())
+	})
+	fmt.Println("}")
+}
+
+var flagsSetup = false
+var flagsMutex = &sync.Mutex{}
+
+func SetupLoggingFlags() {
+	flagsMutex.Lock()
+	if !flagsSetup {
+		if Flags.LogVerbose {
+			// If klog verbosity is not set to a non-default value (via "-args -v=X"),
+			// set up verbosity for klog so round_trippers.go prints:
+			//   URL, request headers, response headers, and partial response body
+			// Definition in vendor/k8s.io/client-go/transport/round_trippers.go:DebugWrappers
+			if flag.CommandLine.Lookup("v").Value.String() == "2" {
+				flag.Set("v", "8")
```

Comment:
Why 8? Where does this constant come from?

Reply:
It's described in the comment above?

Reply:
Not really. It just refences something. It would make it more useful to rewrite as `...v=8 prints...`

Reply:
Still here.

Reply:
The comments describe Why, not How. Rearranged them to make it more obvious.

Reply:
I don't know when I'm supposed to resolve this.

Reply:
I think it's fine, I'd love to see that 8 is what logs that info, but I am not going to beat the bush around this.